cli/cmd/configure_domain.go

```diff
@@ -148,14 +148,6 @@ Example:
 				return err
 			}
 
-			if err := keptnutils.RestartPodsWithSelector(false, "keptn", "run=api"); err != nil {
-				return err
-			}
-
-			if err := keptnutils.WaitForPodsWithSelector(false, "keptn", "run=api", 5, 5*time.Second); err != nil {
-				return err
-			}
-
```

Comment:
Why do we not need to restart the pods anymore?

Reply:
This is not needed anymore.